src/Nncase.Tests/Quant/leaky_relu.quant.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "1.0",
+    "model": "",
+    "outputs": [
+        {
+            "name": "leaky_relu",
+            "data_type": "u8",
```

Comment:
According to Google/Microsoft guideline, we should use camel-casing.
https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format